src/number/index.ts

```diff
@@ -206,3 +206,130 @@ export function millionWithCommas(num: number) {
 export function getIntegerRandomNoBetweenTwoNo(min: number, max: number) { // min, max inclusive
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
+
+
+/**
+ * This method can be used to convert the number in lakhs and crore format.
+ *
+ * @param {number | string} number - Number that you want to format for lakh or crore.
+ * @param {number} toFixed - how much decimal places you need after decimal, default is set as 2.
+ *
+ * @remarks
+ * It returns the number as it is on error ,empty string, null, undefined, NaN is passed.
+ *
+ * @example
+ *
+ * ```
+ * changeFormatToLakhCrore(103213456); // 10.32Cr
+ * changeFormatToLakhCrore('1034564'); // 10.35L
+ * changeFormatToLakhCrore('1027654',3); // 10.277L
+ * ```
+ *
+ */
+export function changeFormatToLakhCrore(num: string | number, toFixedDecimals: number = 2) {
+  try {
+    if (isEmpty(num) || isNaN(num as number)) {
+      console.error('Unable to convert number in lakhs and crores');
+
+      return num;
+    }
+
+    const sign = Math.sign(Number(num));
+    const noOfLakhs = Math.abs(+(Number(num) / 100000));
+
+    if (isNaN(noOfLakhs)) {
+      console.error('NaN error ');
+
+      return num;
+    }
+
+    let displayStr = String(num);
+
+    if (noOfLakhs >= 1 && noOfLakhs <= 99) {
+      const finalLakhs = noOfLakhs * sign;
+      const toFixDisplayVal = finalLakhs.toFixed(toFixedDecimals);
```

Comment:
take this line also out. If case of crores ... you will just have to divide by 100. But test properly.

Reply:
If I take this line also out I have to again check for displayStr to put 'Cr' or 'L' at the end. Better to use only one if condition for this.